.github/ISSUE_TEMPLATE/508-issue.md

```diff
@@ -0,0 +1,50 @@
+---
+name: 508 Accessibility
+about: To describe issues related to Section 508 Accessibility Standards
+title: ''
+labels: '508/Accessibility'
+assignees: ''
+
+---
+
+# Title
+[ERROR TYPE]: Brief description of error
+
+KEYBOARD: Divs styled as buttons ignore Spacebar keypress
+
+## Issue
+This is a detailed description of the issue. It should include a restatement of the issue title, and more background information.
```

Comment:
Do we want these "expectation/informational" statements in this document to be hidden in preview mode (wrapped in `<!--- -->`)?  For example:

```
<!-- This is a detailed description of the issue. It should include a restatement of the issue title, and more background information. -->

Reply:
Yeah, that would be a great idea. I'll update those before merging.